### PR TITLE
sispmctl: init at 4.11

### PR DESCRIPTION
**Pull request by 9R**
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by Aleksanaa** [comment]
Please change commit message and PR title to `sispmctl: init at 4.11`.

File: pkgs/by-name/si/sispmctl/package.nix [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    url = "mirror://sourceforge/${pname}/${pname}-${version}.tar.gz";
+    hash = "sha256-dLlKNxAEaxUHDHMR8MrLgVVMhrQidxnMJzPLlscFJXg=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    libusb-compat-0_1
+  ];
+
+  buildInputs = [ ];
```
Comment (Aleksanaa) [inline pkgs/by-name/si/sispmctl/package.nix:22]: You can remove it if not needed.

File: pkgs/by-name/si/sispmctl/package.nix [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+, pkg-config
+, libusb-compat-0_1
+}:
+
+stdenv.mkDerivation rec {
+  pname = "sispmctl";
+  version = "4.11";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/${pname}/${pname}-${version}.tar.gz";
```
Comment (Aleksanaa) [inline pkgs/by-name/si/sispmctl/package.nix:13]:
```suggestion
    url = "mirror://sourceforge/sispmctl/sispmctl-${version}.tar.gz";
```

File: pkgs/by-name/si/sispmctl/package.nix [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    libusb-compat-0_1
+  ];
+
+  buildInputs = [ ];
+
+  meta = with lib; {
```
Comment (Aleksanaa) [inline pkgs/by-name/si/sispmctl/package.nix:24]: Please add `mainProgram` even if it's the same as `pname`.

Comment (9R) [reply pkgs/by-name/si/sispmctl/package.nix:22]: done

File: pkgs/by-name/si/sispmctl/package.nix (hunk 1 above)
Comment (9R) [reply pkgs/by-name/si/sispmctl/package.nix]: done

File: pkgs/by-name/si/sispmctl/package.nix [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  nativeBuildInputs = [
+    pkg-config
+    libusb-compat-0_1
+  ];
+
+  meta = with lib; {
+    homepage = "https://sispmctl.sourceforge.net/";
+    description = "USB controlled powerstrips management software";
+    license = licenses.gpl2;
+    mainProgram = sispmctl;
```
Comment (sikmir) [inline pkgs/by-name/si/sispmctl/package.nix:26]:
```suggestion
    mainProgram = "sispmctl";
```

File: pkgs/by-name/si/sispmctl/package.nix [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+  ];
+
+  buildInputs = [
+    libusb-compat-0_1
+  ];
+
+  meta = with lib; {
+    homepage = "https://sispmctl.sourceforge.net/";
+    description = "USB controlled powerstrips management software";
+    license = licenses.gpl2;
```
Comment (sikmir) [inline pkgs/by-name/si/sispmctl/package.nix:28]:
```suggestion
    license = licenses.gpl2Plus;
```

File: pkgs/by-name/si/sispmctl/package.nix [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+    mainProgram = "sispmctl";
+    maintainers = [ maintainers._9R ];
+    platforms = platforms.linux;
```
Comment (sikmir) [inline pkgs/by-name/si/sispmctl/package.nix:31]:
```suggestion
    platforms = platforms.unix;
```

File: pkgs/by-name/si/sispmctl/package.nix (hunk 5 above)
Comment (9R) [reply pkgs/by-name/si/sispmctl/package.nix]: thx. changed accordingly

File: pkgs/by-name/si/sispmctl/package.nix (hunk 6 above)
Comment (9R) [reply pkgs/by-name/si/sispmctl/package.nix]: changed as suggested

**Review by fpletz — APPROVED** [review]

**Pull request merged**